examples/fabric-chaincode-example-external/build.gradle

```diff
@@ -0,0 +1,246 @@
+/*
```

Comment:
Great to have an example!  

Think it's worth sliming down the gralde file - it's got a lot of things in it copied from the main build's gradle file. Not needed.